helm/hpcc/templates/_helpers.tpl

```diff
@@ -1477,6 +1480,43 @@ resources:
 {{- end -}}
 {{- end -}}
 
+{{/*
+Add minimal resource allocation for postrun sidecar containers
+
+This template generates low-impact CPU and memory resource specifications
+for utility sidecar containers that perform lifecycle management tasks.
+These containers typically monitor main application containers and perform
+cleanup operations, requiring minimal cluster resources.
+
+The resource values ensure guaranteed scheduling while preventing resource
+contention with primary HPCC Platform workloads (Dali, Thor, Roxie).
+
+Pass in: Current template context (.)
+Returns: YAML resources block with minimal requests and limits
+*/}}
+{{- define "hpcc.addPostrunResources" }}
+resources:
+  requests:
+    cpu: "5m"
+    memory: "8Mi"
+  limits:
+    cpu: "10m"
+    memory: "16Mi"
+{{ end -}}
+
+{{- define "hpcc.addResourcesEx" }}
```

Comment:
this is only here to avoid bug in addResources, this approach could be used to avoid the repeated element name